src/pyspiffe/svid/jwt_svid.py

```diff
@@ -132,7 +132,7 @@ def parse_and_validate(
                     'verify_exp': True,
                 },
             )
-            spiffe_ID = SpiffeId.parse(claims['sub'])
+            spiffe_ID = SpiffeId.parse(claims.get('sub', None))
```

Comment:
nit: [`get()`](https://docs.python.org/3/library/stdtypes.html#dict.get) default value is `None` so it is not necessary to add it here.

Reply:
Thanks!